app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int, *args) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_list: list) -> list[Person]:
+    persons = [Person(temp_person["name"], temp_person["age"])
+               for temp_person in people_list
+               ]
```

Comment:
Bad formatting, brackets start should be same (almost everywhere) as brackets end.
F.e.:
[something,
something]
or
[
something,
something
]